server/v2/api/grpcgateway/interceptor.go

```diff
@@ -41,57 +57,77 @@ func newGatewayInterceptor[T transaction.Tx](logger log.Logger, gateway *runtime
 	if err != nil {
 		return nil, err
 	}
+	// convert the mapping to regular expressions for URL matching.
+	wildcardMatchers, simpleMatchers := createRegexMapping(logger, getMapping)
+	if err != nil {
```

Comment:
might rm this err check

Reply:
oops, thanks - removed it in [remove unncessary error check](https://github.com/cosmos/cosmos-sdk/pull/23262/commits/db043fe1a25277199fb036e5d5a8dcd0c991d5fc)